server/config/config.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Flow Playground
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+	"log"
+)
+
+// config holds parsed environment variables
+var config struct {
+	platform   PlatformConfig
+	playground PlaygroundConfig
+	sentry     SentryConfig
+	database   DatabaseConfig
+}
+
+// init parses all environment variables into config
+func init() {
+	config.platform.getConfig()
+	config.playground.getConfig()
+	config.sentry.getConfig()
+	config.database.getConfig()
+}
+
+func Platform() PlatformType {
+	return config.platform.Type
+}
+
+func Playground() PlaygroundConfig {
+	return config.playground
+}
+
+func Sentry() SentryConfig {
+	return config.sentry
+}
+
+func Database() DatabaseConfig {
+	return config.database
+}
+
+// envConfig interface for sub-configs
+type envConfig interface {
```

Comment:
does this conflict with the import? maybe better to reanme it. For interface it's good to have "-er" names, so it could be "configGetter"

Reply:
Also where is this interface used?

Reply:
Sounds good, renamed it. This is used for each config type: Database, Playground, Platform, Sentry.